cli.js

```diff
@@ -158,7 +158,10 @@ function exit(results) {
 	// correctly flush the output when multiple test files
 	process.stdout.write('');
```

Comment:
@vdemedes, this line is from #47 

Can you remember what you were addressing there? Tests pass without it. (even before my changes).